src/main/java/uk/gov/ons/census/casesvc/model/dto/ResponseDTO.java

```diff
@@ -16,7 +16,8 @@
 
   private String questionnaireId;
 
-  private boolean unreceipt;
+  @JsonInclude(Include.NON_NULL)
```

Comment:
could be put at class level requiring single annotation

Reply:
Done 👀